src/main/java/cart/domain/BasicDeliveryFeeCalculator.java

```diff
@@ -0,0 +1,18 @@
+package cart.domain;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class BasicDeliveryFeeCalculator implements DeliveryFeeCalculator{
+    
+    public static int BASIC_FEE = 3000;
```

Comment:
```suggestion
    public static int BASIC_FEE = 3_000;
```
이런식으로 하면 보기가 더 좋은거 같아요.

Reply:
수정했습니다